00_make_block_group_shp.R

```diff
@@ -67,5 +82,9 @@ tracts_sf_1970 <- tracts_sf_1970 %>%
 
 saveRDS(tracts_sf_1970, "tracts_1970_5072.rds")
 
+tracts_sf_1970 <- s3::s3_get("s3://geomarker/geometries/tracts_1970_5072.rds") %>% readRDS()
+tracts_sf_1970 <- sf::st_make_valid(tracts_sf_1970)
+saveRDS(tracts_sf_1970, "tracts_1970_5072.rds")
```

Comment:
you don't need to get files from s3, right?  they are read in locally from the shp files...

I'm assuming this is how you updated the files to make them valid? But we should code in how it would be done "from scratch":

- make the `st_read` filepaths relative to the working directory, so we don't have to include your username/Downloads... etc
- put comment in R code where you uploaded the rds files to S3

Reply:
That makes sense. Sometimes I'm not sure if I should keep code exactly how I did it or clean it up, but I don't think there should be any conflicts here with the cleaned up workflow.